examples/fuse_examples/imaging/classification/knight/baseline/dataset.py

```diff
@@ -1,198 +1,264 @@
+        (OpPrepare_Clinical(), dict()), #process clinical data
+
+    ])
 
-    augmentation_pipeline = [
-        [
-            ("data.input.image",),
-            rotation_in_3d,
-            {
-                "z_rot": Uniform(-5.0, 5.0),
-                "y_rot": Uniform(-5.0, 5.0),
-                "x_rot": Uniform(-5.0, 5.0),
-            },
-            {"apply": RandBool(0.5)},
-        ],
-        [("data.input.image",), squeeze_3d_to_2d, {"axis_squeeze": "z"}, {}],
-        [
-            ("data.input.image",),
-            aug_op_affine,
-            {
-                "rotate": Uniform(0, 360.0),
-                "translate": (RandInt(-14, 14), RandInt(-14, 14)),
-                "flip": (RandBool(0.5), RandBool(0.5)),
-                "scale": Uniform(0.9, 1.1),
-            },
-            {"apply": RandBool(0.9)},
-        ],
-        [
-            ("data.input.image",),
-            aug_op_gaussian,
-            {"std": 0.01},
-            {"apply": RandBool(0.9)},
-        ],
-        [
-            ("data.input.image",),
-            unsqueeze_2d_to_3d,
-            {"channels": 1, "axis_squeeze": "z"},
-            {},
-        ],
-    ]
+    val_dynamic_pipeline = PipelineDefault("dynamic", [
+                
+        # resize image to (110, 256, 256)
+        # (OpLambda(func=partial(my_resize, resize_to=(110, 256, 256))), dict(key="data.input.img")),
+
+        # Numpy to tensor
+        (OpToTensor(), dict(key="data.input.img")),
+        (OpToTensor(), dict(key="data.input.clinical.all")),
+
+        
+        (OpRandomCrop(), dict(key="data.input.img", out_size=resize_to, centralize=True)),
+        # (OpLambda(lambda x: aug_op_random_crop_and_pad(x, resize_to, centralize=False)), dict(key="data.input.img")),
+        # add channel dimension -> [C=1, D, H, W]
+        (OpLambda(lambda x: x.unsqueeze(dim=0)), dict(key="data.input.img")),  
+    ]) 
+
+    train_dynamic_pipeline = PipelineDefault("dynamic", [
+                
+        # resize image to (110, 256, 256)
+        # (OpLambda(func=partial(my_resize, resize_to=(110, 256, 256))), dict(key="data.input.img")),
+
+        # Numpy to tensor
+        (OpToTensor(), dict(key="data.input.img")),
+        (OpToTensor(), dict(key="data.input.clinical.all")),
+
+        # affine transformation per slice but with the same arguments
+        (OpRandApply(OpSample(OpAugAffine2D()), 0.5) , dict(
+            key="data.input.img",
+            rotate=Uniform(-180.0,180.0),
+            scale=Uniform(0.8, 1.2),
+            flip=(RandBool(0.5).sample(), RandBool(0.5)),
+            translate=(RandInt(-15, 15).sample(), RandInt(-15, 15))
+        )),
+        (OpRandomCrop(), dict(key="data.input.img", out_size=resize_to)),
+        # (OpLambda(lambda x: aug_op_random_crop_and_pad(x, resize_to, centralize=False)), dict(key="data.input.img")),
```

Comment:
remove commented out code (in all other places as well)

Reply:
done